osgp/platform/osgp-adapter-domain-core/src/test/java/org/opensmartgridplatform/adapter/domain/core/application/services/DefaultDeviceResponseServiceTest.java

```diff
@@ -0,0 +1,139 @@
+package org.opensmartgridplatform.adapter.domain.core.application.services;
```

Comment:
indentation is not ok, check your IDE settings

Reply:
I should now have properly have set up my IDE settings

Reply:
I still see its not ok.

Reply:
still not ok

Reply:
I now manually ran the auto code format in the eclipse workspace. If it now isn't correct I'll need to have a look at my eclipse set up.